sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1588,19 +1594,63 @@ public void setProfilesSampleRate(final @Nullable Double profilesSampleRate) {
   /**
    * Returns a list of origins to which `sentry-trace` header should be sent in HTTP integrations.
    *
+   * @deprecated use {{@link SentryOptions#getTracePropagationTargets()} }
    * @return the list of origins
    */
+  @Deprecated
+  @SuppressWarnings("InlineMeSuggester")
   public @NotNull List<String> getTracingOrigins() {
-    return tracingOrigins;
+    return getTracePropagationTargets();
   }
 
   /**
    * Adds an origin to which `sentry-trace` header should be sent in HTTP integrations.
    *
+   * @deprecated use {{@link SentryOptions#addTracePropagationTarget(String)} }
    * @param tracingOrigin - the tracing origin
    */
+  @Deprecated
+  @SuppressWarnings("InlineMeSuggester")
   public void addTracingOrigin(final @NotNull String tracingOrigin) {
-    this.tracingOrigins.add(tracingOrigin);
+    addTracePropagationTarget(tracingOrigin);
+  }
+
+  @Deprecated
+  @SuppressWarnings("InlineMeSuggester")
+  @ApiStatus.Internal
+  public void setTracingOrigins(List<String> tracingOrigins) {
+    setTracePropagationTargets(tracingOrigins);
+  }
+
+  /**
+   * Returns a list of origins to which `sentry-trace` header should be sent in HTTP integrations.
+   *
+   * @return the list of targets
+   */
+  public @NotNull List<String> getTracePropagationTargets() {
+    if (tracePropagationTargets == null) {
+      return defaultTracePropagationTargets;
+    }
+    return Collections.unmodifiableList(tracePropagationTargets);
+  }
+
+  @ApiStatus.Internal
+  public void setTracePropagationTargets(List<String> tracePropagationTargets) {
```

Comment:
```suggestion
  public void setTracePropagationTargets(final List<String> tracePropagationTargets) {
```
also, can it be nullable?

Reply:
Might make sense to allow passing `null` to restore the default regex.